Snakefile

```diff
@@ -429,6 +429,17 @@ def terminate_if_cutout_exists(config=config):
     Check if any of the requested cutout files exist.
     If that's the case, terminate execution to avoid data loss.
     """
+    import argparse
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--mode", type=int, default=snakemake.Mode.default)
+    parser.add_argument("--target-jobs", type=str, nargs="+", default=[])
+    args, _ = parser.parse_known_args(sys.argv)
+    if args.mode == snakemake.Mode.subprocess and "build_cutout" not in " ".join(
+        args.target_jobs
+    ):
+        return
+
```

Comment:
this fixes the issue with the build cutout termination after it has already been built. I've made a PR upstream as well https://github.com/pypsa-meets-earth/pypsa-earth/pull/1079